### PR TITLE
Fix autofix suggestions for LevelCrossingOnRailwayCheck

### PR DESCRIPTION
**Pull request by atiannicelli**
Description:
Fixed the auto-fix suggestions for level crossing on railway check. The old auto-fix suggestion suggested to the user to remove the railway=level_crossing tag if the node was not a railway crossing. The new auto-fix suggestion detects if there are pedestrian ways going through this node or a cycleway. If that is the case then the new suggestion will be to change the railway=level_crossing tag to railway=crossing.

Potential Impact:
Changes the autofix suggestion for some of the issues found in the level Crossing on Railway check.

Unit Test Approach:
Created challenge from ARG, CHL, and NZL and verified that the new suggestions looked correct.
ARG - https://maproulette.org/challenge/15923
CHL - https://maproulette.org/challenge/16429
NZL -  https://maproulette.org/challenge/16430

Here is an example of the new suggestion:
1) https://maproulette.org/challenge/16428/task/80753848
2) https://maproulette.org/challenge/16428/task/80753740
3) https://maproulette.org/challenge/16428/task/80751535

Test Results:
Development testing - I ran through the new cases like the ones above and made sure they look good to me.

Country | # Required Review | # FP | % FP Rate
-- | -- | -- | --
Argentina | 600 | 15 | 2.5%
Chile | 300 | 3 | 1%
New Zealand | 100 | 9 | 9%

### CODE REVIEW

**Review by seancoulter — DISMISSED** [review]
Looks good, thanks for adding all the new instructions!

**Comment by Bentleysb** [comment]
This looks good to me. Looks like there is one code small to clean up. I also wanted to check on the FP rate. Is that for the check as a whole, or for just the fix suggestions?

**Comment by atiannicelli** [comment]
Those metrics were for the check as a whole.

I'll do something about the code smell... although I like it as is (easier to understand/read)

File: src/main/java/org/openstreetmap/atlas/checks/validation/intersections/LevelCrossingOnRailwayCheck.java [hunk 1]
```diff
@@ -55,45 +57,51 @@
...
+            + "To fix: replace the `railway=crossing` tag with a `railway=level_crossing` tag";
+    private static final int NODE_CROSSING_MISSING_LC_TAG_INDEX = 1;
+    private static final String NODE_MISSING_LC_TAG = "The intersection node (OSM ID: {0,number,#}) is missing a `railway=level_crossing` tag. "
+            + "This means that there are at least one valid railway and one car navigable highway on the same layer at this node. "
+            + "To fix: If the two ways should be on different layers then adjust the layer tags for each way appropriately. "
+            + "If the two ways do intersect on the same layer then add the `railway=level_crossing` tag to this node.";
+    private static final int NODE_MISSING_LC_TAG_INDEX = 2;
+    private static final String NODE_INVALID_LC_TAG_NO_HIGHWAY = "The node (OSM ID: {0,number,#}) has `railway=level_crossing` tag, but there is no car navigable highway at this intersection. "
+            + "To fix: Verify that no highway crosses the railway at this node on the same layer and tehn remove `railway=level_crossing` tag.";
```
Comment (Bentleysb) [inline src/main/java/org/openstreetmap/atlas/checks/validation/intersections/LevelCrossingOnRailwayCheck.java:81]: typo: tehn -> then

**Review by seancoulter — APPROVED** [review]
Looks good, just that typo ^

**Comment by ameliaewatts** [comment]
@atiannicelli How's this coming along? I think we want to try to get this into the release today.

**Comment by atiannicelli** [comment]
Just pushed the typo fix. I think it is ready if everyone is good with it.

**Review by Bentleysb — APPROVED** [review]

**Review by seancoulter — APPROVED** [review]

**Pull request merged**